src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -79,7 +79,7 @@ const Body = BaseForm.extend({
       return request({
         url: getAuthenticatorUrl('challenge'),
         method: 'POST',
-        body: JSON.stringify({ challengeRequest }),
+        data: JSON.stringify({ challengeRequest }),
```

Comment:
🤦‍♂

Reply:
guess this is needed because we are using jquery to make the post ?

Reply:
The wrong key was used before, thus no data was sent in the body.

Reply:
yeah, I wonder if its because we are using jquery.ajax. I see that we set `body` in [httpClient](https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/httpClient.js#L28) that internally uses [cross-fetch](https://github.com/lquixada/cross-fetch). Additional [fetch API ](https://github.github.io/fetch/) doc